MSAL/src/MSALPublicClientApplication.m

```diff
@@ -584,7 +593,8 @@ - (void)acquireTokenWithParameters:(MSALInteractiveTokenParameters *)parameters
     };
     
     MSIDDefaultTokenRequestProvider *tokenRequestProvider = [[MSIDDefaultTokenRequestProvider alloc] initWithOauthFactory:self.msalOauth2Provider.msidOauth2Factory
-                                                                                                          defaultAccessor:_tokenCache
+                                                                                                          defaultAccessor:self.tokenCache
+                                                                                                  accountMetadataAccessor:self.accountMetadataCache
```

Comment:
theoretically it doesn't need to be passed through token request provider, although I don't see an issue with it

Reply:
discussed offline, there might be a way to better define controller and tokenRequestProvider, but for now this is ok